README.md

```diff
@@ -2098,14 +2115,7 @@ set :protection, :except => [:path_traversal, :session_hijacking]
 ```
 
 By default, Sinatra will only set up session based protection if `:sessions`
-has been enabled. Sometimes you want to set up sessions on your own, though. In
-that case you can get it to set up session based protections by passing the
-`:session` option:
-
-```ruby
-use Rack::Session::Pool
-set :protection, :session => true
-```
+have been enabled. See 'Using Sessions'.
```

Comment:
Oh I see what you mean. That's the weird thing, I can't figure out how that example code would have ever did what it claimed to do!